package.json

```diff
@@ -22,7 +22,9 @@
   "license": "MIT",
   "dependencies": {
     "ember-auto-import": "^1.2.15",
+    "ember-class-based-modifier": "^0.10.0",
     "ember-cli-babel": "^7.7.3",
+    "fast-deep-equal": "^2.0.1",
```

Comment:
💯 Going to pull this into [intersection-observer-admin](https://github.com/snewcomer/intersection-observer-admin)